packages/pigeon/lib/java_generator.dart

```diff
@@ -534,9 +555,15 @@ class JavaGenerator extends StructuredGenerator<JavaOptions> {
           : _javaTypeForDartType(method.returnType);
       argSignature.add('Result<$resultType> result');
     }
-    addDocumentationComments(
-        indent, method.documentationComments, _docCommentSpec);
-
+    if (method.documentationComments.isNotEmpty) {
+      addDocumentationComments(
+          indent, method.documentationComments, _docCommentSpec);
+    } else {
+      indent.newln();
+    }
+    if (nullableType != '') {
+      indent.writeln(nullableType);
```

Comment:
Isn't this also a functional change?

Reply:
No, I just separated the nullability onto a new line. The same code gets written.